test/integration/helpers.js

```diff
@@ -470,6 +470,7 @@ async function runMochaWatchJSONAsync(args, opts, change) {
       // eslint-disable-next-line no-control-regex
       .replace(/\u001b\[\?25./g, '')
       .split('\u001b[2K')
+      .filter(x => x)
```

Comment:
I filtered output because the output is an empty string(`''`) when a test file is crashed.